language-design.md

```diff
@@ -28,7 +77,33 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+The most important feature of a well designed language in our opinion is
+understandability. Steele mentions that including too many features for
+specialized groups of users would "just be too much" [Steele, 2012]. This
+is in line with understandability because users don't need to learn as 
+many features. This is also in line with knowing the intended audience for
+a language and catering to their needs - Java is a general purpose language
+and therefore should not be too specialized for any one field. 
+
+The syntax for a well designed language should be intuitive. It has to mesh
+well with the native language of users because language dictates how users
+think. According to Andreas Stefik, "usability testing showed that simply 
+finding natural-language replacements for some of the more abstruse syntax 
+went a long way" [Pavlus, 2012]. One example is Racket's syntax for addition,
+for example, is not intuitive to English speakers because the operator is
+first followed by the two numbers to be added. 
```

Comment:
It's also hard if you consider that all of these articles assume English as the natural language, which I think is already problematic in languages without natural language features.